docs/UserGuide.md

```diff
@@ -478,7 +477,7 @@ The index **must be a positive integer** 1, 2, 3, …​
 
 
 **Examples**:
-* `listt` followed by `deletet 1` lists out all the tasks in NUS Classes, then deletes the task at index 1.
+* `listt all/` followed by `deletet 1` lists out all the tasks in NUS Classes, then deletes the task at index 1.
```

Comment:
Good catch, is there any more incorrect usages of listt throughout?

Reply:
Should be no more. But just updated the command summary listt